prime-router/src/main/kotlin/history/azure/HistoryDatabaseAccess.kt

```diff
@@ -72,23 +75,38 @@ abstract class HistoryDatabaseAccess(
         klass: Class<T>,
         reportId: UUID? = null,
         fileName: String? = null,
+        receivingOrgSvcStatus: String? = null,
     ): List<T> {
         val sortedColumn = createColumnSort(sortColumn, sortDir)
-        val whereClause = createWhereCondition(organization, orgService, reportId, fileName, since, until, showFailed)
+        val whereClause =
+            createWhereCondition(
+                organization, orgService, receivingOrgSvcStatus, reportId, fileName, since, until, showFailed
+            )
 
         return db.transactReturning { txn ->
             val query = DSL.using(txn)
                 // Note the report file and action tables have columns with the same name, so we must specify what we need.
                 .select(
                     ACTION.ACTION_ID, ACTION.CREATED_AT, ACTION.SENDING_ORG, ACTION.SENDING_ORG_CLIENT,
                     REPORT_FILE.RECEIVING_ORG, REPORT_FILE.RECEIVING_ORG_SVC,
+                    DSL.jsonbGetAttributeAsText(SETTING.VALUES, "customerStatus").`as`("receiving_org_svc_status"),
```

Comment:
Nit: does this value need to get returned?

Reply:
Yes. If we pass `active` then both `active` and `testing` status will get returned and we may want a way to view this in the UI.